Ginger/GingerCore/Actions/ActButton.cs

```diff
@@ -110,7 +110,7 @@ ePlatformType IObsoleteAction.GetTargetPlatform()
 
         bool IObsoleteAction.IsObsoleteForPlatform(ePlatformType platform)
         {
-            if (platform == ePlatformType.Web || platform == ePlatformType.Mobile || platform == ePlatformType.NA)
+            if (platform == ePlatformType.Web || platform == ePlatformType.NA)
             {
                 return true;
             }
```

Comment:
i asked to remove the Mobile only for GenericElement Action and leave it for rest